.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,65 @@
+# Copied from NumPy https://github.com/numpy/numpy/pull/25894
+# https://github.com/numpy/numpy/blob/d2d2c25fa81b47810f5cbd85ea6485eb3a3ffec3/.github/workflows/emscripten.yml
+#
+
+name: Test Emscripten/Pyodide build
+
+on: [push, pull_request, merge_group]
+
+env:
+  FORCE_COLOR: 3
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  build-wasm-emscripten:
+    name: Build scikit-image distribution for Pyodide
+    runs-on: ubuntu-22.04
+    env:
+      PYODIDE_VERSION: 0.26.0
+      # PYTHON_VERSION and EMSCRIPTEN_VERSION are determined by PYODIDE_VERSION.
+      # The appropriate versions can be found in the Pyodide repodata.json
+      # "info" field, or in Makefile.envs:
+      # https://github.com/pyodide/pyodide/blob/main/Makefile.envs#L2
+      PYTHON_VERSION: 3.12.1
+      EMSCRIPTEN_VERSION: 3.1.58
+      NODE_VERSION: 20
+    steps:
+      - name: Checkout scikit-image
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        id: setup-python
+        uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # v5.0.0
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Set up Emscripten toolchain
+        uses: mymindstorm/setup-emsdk@6ab9eb1bda2574c4ddb79809fc9247783eaf9021 # v14
+        with:
+          version: ${{ env.EMSCRIPTEN_VERSION }}
+          actions-cache-folder: emsdk-cache
+
+      - name: Install pyodide-build
+        run: pip install pyodide-build==${{ env.PYODIDE_VERSION }}
+
+      - name: Build scikit-image for Pyodide
+        run: |
+          pyodide build
```

Comment:
This approach is totally fine, just wondering about the advantage of the manual vs cibuildwheel approach?

Reply:
I am happy to implement `cibuildwheel`, but it was very recently merged (https://github.com/pypa/cibuildwheel/pull/1456) and there hasn't been a release yet. I did try it for NumPy (https://github.com/numpy/numpy/pull/26570), pandas (https://github.com/pandas-dev/pandas/pull/58647), and PyWavelets (https://github.com/PyWavelets/pywt/pull/744), and all of them work – so if it would be okay to use using `pypa/cibuildwheel@main` for now and merge, I am happy to implement the changes as necessary.